src/samplers/ratelimiting_sampler.js

```diff
@@ -32,7 +32,7 @@ export default class RateLimitingSampler {
         }
 
         this._maxTracesPerSecond = maxTracesPerSecond;
-        this._rateLimiter = new RateLimiter(maxTracesPerSecond);
+        this._rateLimiter = new RateLimiter(maxTracesPerSecond, 2.0);
```

Comment:
but the previous behavior was `this._balance > this._creditsPerSecond`, so wouldn't it be backwards compatible to use `new RateLimiter(maxTracesPerSecond, maxTracesPerSecond)`? Otherwise if maxTracesPerSecond=10, you can never actually achieve that limit since max balance is just 2